lib/dom-parser.js

```diff
@@ -1,8 +1,14 @@
 var conventions = require("./conventions");
 var entities = require('./entities');
 
+var DOMImplementation = require('./dom').DOMImplementation;
+var sax = require('./sax');
+
 var NAMESPACE = conventions.NAMESPACE;
 
+var XMLReader = sax.XMLReader;
+var ParseError = sax.ParseError;
```

Comment:
nit: Again just alphabetical order of "imports"
```suggestion
var ParseError = sax.ParseError;
var XMLReader = sax.XMLReader;
```